content/en/docs/upgrade/upgrade-with-kubekey.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Upgrade with KubeKey"
+keywords: "kubernetes, upgrade, kubesphere, v3.0.0"
+description: "Upgrade KubeSphere with kubekey"
+
+linkTitle: "Upgrade with KubeKey"
+weight: 100
+---
+KubeKey is recommended for users whose Kubernetes was deployed together with KubeSphere.
```

Comment:
KubeKey is recommended for users whose KubeSphere and Kubernetes were both deployed by KubeSphere Installer. If your Kubernetes cluster was provisioned by yourself, please refer to [Upgrade with ks-installer](xxx).